README.md

```diff
@@ -1 +1,19 @@
 # api-snippets
+
+## Usage
+Install the node dependencies:
+
+```bash
+  npm install
+```
+
+Build version 1.0 of api snippets:
+
+```bash
+  npm run buildv1.0
+```
+
+## Questions
+
+1) Are paths in the Mojaloop Specification considered "snippets"? Are they re-used widely across services?
+2) Do we need to support both Swagger 2.0 and Open Api 3?
```

Comment:
I would suggest to stick only to open-api-v3

Reply:
Ok. I think thats best for now unless another stakeholder raises their voice for 2.0.